Extensions/Terraform/Src/Tasks/TerraformTaskV1/task.json

```diff
@@ -0,0 +1,318 @@
+{
+    "id": "FE504ACC-6115-40CB-89FF-191386B5E7BF",
+    "name": "TerraformCustomTask",
+    "friendlyName": "Custom Terraform Task",
+    "description": "Execute terraform cli commands",
+    "author": "Akash Chakrabarti",
+    "helpMarkDown": "",
+    "category": "Deploy",
+    "visibility": [
+        "Release"
+    ],
+    "demands": [
+        "terraform"
+    ],
+    "version": {
+        "Major": "0",
+        "Minor": "0",
+        "Patch": "85"
+    },
+    "instanceNameFormat": "$(provider) - terraform $(command)",
+    "execution": {
+        "Node": {
+            "target": "src/index.js"
+        }
+    },
+    "groups": [
+        {
+            "name": "backendAzureRm",
+            "displayName": "AzureRM Backend Configuration",
+            "isExpanded": false,
+            "visibleRule": "provider = azurerm && command = init && backendTypeAzureRM = azurerm"
+        },
+        {
+            "name": "backendAWS",
```

Comment:
@RoopeshNair  - Feedback from @shashankbarsin . Should we take the state file location as task input or just take it from config file?